nixos/modules/services/networking/pihole-ftl.md

```diff
@@ -0,0 +1,53 @@
+# pihole-FTL {#module-services-networking-pihole-ftl}
+
+*Upstream documentation*: <https://docs.pi-hole.net/ftldns/>
+
+pihole-FTL is a fork of [Dnsmasq](#module-services-networking-dnsmasq),
```

Comment:
```suggestion
pihole-FTL is a fork of {#module-services-networking-dnsmasq},
```

Reply:
I think it's because I pulled the Dnsmasq manual changes out into #275902. The reference once exist until that PR is merged, I'll wait until that's done and apply your suggestions

Reply:
That PR is merged, and this is resolved